cmd/tracker/capture.go

```
@@ -0,0 +1,76 @@
+	bundleID string
+}
+
+// recordExecutedSpec stores the spec for the run about to start.
+func recordExecutedSpec(path, source string, workflow *ir.Workflow) {
+	capturedSpec.path = path
+	capturedSpec.source = source
+	capturedSpec.workflow = workflow
+}
+
+// recordBundleIdentity stores the .dipx identity for a bundle run.
+func recordBundleIdentity(id string) {
```

golangci-lint (2.12.2)

[error] func recordBundleIdentity is unused

(unused)